src/crimson/osd/pg_recovery.cc

```diff
@@ -26,6 +26,8 @@ using std::map;
 using std::set;
 using PglogBasedRecovery = crimson::osd::PglogBasedRecovery;
 
+std::atomic<uint64_t> PGRecovery::ongoing_pushes = 0;
```

Comment:
Why does this need to be atomic?

Reply:
Sorry, that's a mistake, will correct it.